README.md

```diff
@@ -14,27 +14,74 @@ You can deploy these samples directly through the Azure Portal or by using the s
 
 To deploy a sample using the Azure Portal, click the **Deploy to Azure** button found in the README.md of each sample.
 
-To deploy the sample via the command line (using [Azure PowerShell or the Azure CLI](https://azure.microsoft.com/en-us/downloads/)) you can use the scripts.
+To deploy the sample via the command line (using [Azure PowerShell](https://docs.microsoft.com/en-us/powershell/azure/overview) or the [Azure CLI 1.0](https://docs.microsoft.com/en-us/azure/cli-install-nodejs)) you can use the scripts below.
 
-Simply execute the script and pass in the folder name of the sample you want to deploy.  For example:
+Simply execute the script and pass in the folder name of the sample you want to deploy.  
 
+For example:
+
+### PowerShell
 ```PowerShell
 .\Deploy-AzureResourceGroup.ps1 -ResourceGroupLocation 'eastus' -ArtifactStagingDirectory '[foldername]'
 ```
+
+### Bash
 ```bash
 azure-group-deploy.sh -a [foldername] -l eastus
 ```
+
+### Bash on Ubuntu on Windows
+
+Run the following the first time to get all prerequisites installed:
```

Comment:
shouldn't the prereq install be doc'd on ms.com?  (we don't list the PowerShell prereqs)